docs/upload.md

```diff
@@ -24,6 +24,10 @@ best practice to minimise the risk of personally identifiable information being
 The upload command performs metadata validation and client-side removal of human reads for each of your samples, 
 before uploading sequences to EIT Pathogena for analysis.
 
+If trying to upload files in fastq.gz format it is worth cloning the [gpas-user-tools](https://github.com/GlobalPathogenAnalysisService/gpas-user-tools) repo. Following the documentaion, the upload.csv file required for the pathogena upload command can be generated. 
+
+Once the upload.csv file is generated it can be used with the above upload command. Note: the upload csv must be located in the same directory as the sample.fastq.gz files. 
+
```

Comment:
There is a `pathogena build-csv` sub-command that will generate a CSV from a given directory of files and an [example CSV in the assets section](https://github.com/EIT-Pathogena/client/blob/main/docs/assets/example-input.csv), maybe we need to signpost those.

Reply:
Oh yeah, definitely! I don't think anyone from our side found that in the docs